engine/src/main/java/org/terasology/engine/rendering/nui/layers/mainMenu/savedGames/GameProvider.java

```diff
@@ -113,16 +113,16 @@ public static String getNextGameName() {
      * game.
      *
      * <pre>
-     *     1. "Game"    -> "Game"
-     *     2. "Game"    -> "Game 1"
-     *     3. "Game 1"  -> "Game 2"
```

Comment:
Not even here (in `<pre>`)? 😔 Maybe needs another `{@code ...` or so, never understood how that really works...